packages/client/src/context/movieListContext.js

```diff
@@ -19,7 +19,7 @@ export const MovieListProvider = ({ children }) => {
   useEffect(() => {
     const fetchMovies = async () => {
       try {
-        let url = `${apiURL()}/movies?sortBy=${sortBy}`;
+        let url = `${apiURL()}/movies?sortBy=${sortBy}&isFavoritePage=${isFavoritePage}&userId=${3}`;
```

Comment:
You don't need to wrap the 3 in curly braces. Though, how come it's hard-coded to a specific user? Wouldn't it be better to define the value as some sort of parameter?

Reply:
Once the userContext is available, we will refactor it. Thanks for the review